src/Compilers/CSharp/Test/Semantic/Semantics/NullableContextTests.cs

```diff
@@ -21,6 +21,26 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics
 {
     public class NullableContextTests : CSharpTestBase
     {
+        private static readonly NullableDirectives[] s_nullableDirectives = new[]
```

Comment:
>s_nullableDirectives [](start = 53, length = 20)

Consider moving this immediately before `AnalyzeMethodsInEnabledContextOnly_01_Data1()`.